more_itertools/more.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import print_function
+from __future__ import division, print_function
```

Comment:
I couldn't remember whether `//` was a first-class citizen in Py2.7; it seems it is.